core/api/src/main/java/org/eclipse/sensinact/core/command/ResourceCommand.java

```diff
@@ -39,7 +45,7 @@ public ResourceCommand(String model, String provider, String service, String res
     @Override
     protected final Promise<T> call(SensinactDigitalTwin twin, PromiseFactory pf) {
         SensinactResource r = model == null ? twin.getResource(provider, service, resource)
-                : twin.getResource(model, provider, service, resource);
+                : twin.getResource(packageUri, model, provider, service, resource);
```

Comment:
```suggestion
                : packageUri == null ? twin.getResource(model, provider, service, resource)
                : twin.getResource(packageUri, model, provider, service, resource);
```

Reply:
This doesn't really makes a difference, as it again is checked in the ModelNexus.